org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/acct/WAcctViewer.java

```diff
@@ -799,9 +807,26 @@ private void actionExport() {
 			RModelExcelExporter exporter = new RModelExcelExporter(m_rmodel);
 			File file;
 			try {
-				file = new File(FileUtil.getTempMailName(Msg.getMsg(Env.getCtx(), TITLE), ".xls"));
+				file = new File(FileUtil.getTempMailName(Msg.getMsg(Env.getCtx(), TITLE), ".xlsx"));
 				exporter.export(file, Env.getLanguage(Env.getCtx()));
-				Filedownload.save(file, "application/vnd.ms-excel");
+				AMedia media = new AMedia(file.getName(), null, Medias.EXCEL_MIME_TYPE, file, true);
+				IMediaView view = Extensions.getMediaView(Medias.EXCEL_MIME_TYPE, Medias.EXCEL_FILE_EXT, ClientInfo.isMobile());
```

Comment:
shouldn't the two be using Medias.EXCEL_XML_MIME_TYPE and Medias.EXCEL_XML_FILE_EXT instead ?